next.config.js

```diff
@@ -4,6 +4,7 @@ const withPWA = require('next-pwa');
 module.exports = withPWA({
   pwa: {
     dest: 'public',
+    disable: process.env.NODE_ENV === 'development',
     register: true,
     skipWaiting: true,
     runtimeCaching: true,
```

Comment:
```suggestion
    skipWaiting: true,
    runtimeCaching: true,
```

`register` is true by default